swephp.c

```diff
@@ -2517,6 +2515,8 @@ Get the name of a house system
 =head3 return array
 
       retflag => (int)            ERR or eclipse type
+		  There are the following eclipse types for lunar eclipses:
+		  SE_ECL_TOTAL, SE_ECL_PARTIAL, SE_ECL_PENUMBRAL
```

Comment:
The docs list the following eclipse types for `swe_lun_occult_where()` function:
```
SE_ECL_TOTAL
SE_ECL_ANNULAR
SE_ECL_TOTAL | SE_ECL_CENTRAL
SE_ECL_TOTAL | SE_ECL_NONCENTRAL
SE_ECL_ANNULAR | SE_ECL_CENTRAL
SE_ECL_ANNULAR | SE_ECL_NONCENTRAL
SE_ECL_PARTIAL
```
I assume there is a reason I'm unaware of which replaces the missing ones with `SE_ECL_PENUMBRAL` instead?

It's also just a doc block so not a significant concern, simply something I noticed.